example/lib/presentation/pages/calendar_event.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:io';
 
 import 'package:collection/collection.dart';
 import 'package:device_calendar/device_calendar.dart';
+import 'package:device_calendar_example/presentation/pages/color_picker_dialog.dart';
```

Comment:
Tests are failing because of this import. Looking at your directory layout it should be
```dart
import 'package:device_calendar_example/presentation/color_picker_dialog.dart';
```

Reply:
Sorry, issue should now be fixed.